bootstrap-ci-orbit.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Install required dependencies
+# Dev dependencies:
+# - cmake >= 3.15
+# - ninja (optional)
+# - bison (for compiling qt5)
+# - flex (for compiling qt5)
+
+# Load Submodules (vcpkg, libunwindstack)
+git submodule update --init --recursive
+
+if [ $? -ne 0 ]; then
+  echo "Orbit: Could not update/initialize all the submodules. Exiting..."
+  exit 1
+fi
+
+# Build vcpkg
+cd external/vcpkg
+
+# Copy the prebuild binaries from user disk
+cp -a /vcpkg/scripts scripts
+cp -a /vcpkg/downloads downloads
+cp -a /vcpkg/installed installed
+cp /vcpkg/vcpkg vcpkg
+mkdir buildtrees
+cp -a /vcpkg/buildtrees/freetype-gl buildtrees/freetype-gl
+
```

Comment:
from here it seems like this is a copy of linux versoin of the script? can this be reused?

Reply:
Hmm, good point. Looking at that part again: The first part is useless because the vcpkg binary is always there. The second part is useless since to will do nothing unless the list of packages changes - and that would be done in the "normal" bootstrap script. 
I think is is cleaner to just fail if any dependencies change. So I removed all the calls to vcpkg.